### PR TITLE
Reportback cropping fixes

### PR DESCRIPTION
**Pull request by sbsmith86**
This branch fixes a few bugs I found when testing cropping and fixes some of the styling. 
- Switches the rotate and change photo buttons to be tertiary neue buttons with an override for spacing.
- Changes the done button in the modal to be the neue primary button. 
- Initiates crop after the preview image src has been added to the page to make sure it has an image to initiate on.

@DFurnes @weerd /cc @barryclark

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss [hunk 1]
```diff
@@ -7,22 +7,19 @@
...
-    a {
-      color: $off-black;
-      text-decoration: underline;
-    }
-
     .__buttons {
       text-align: right;
 
-      .-rotate {
+      * {
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss]: Yikes! What's going on here?

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss [hunk 2]
```diff
@@ -31,17 +28,15 @@
...
   }
 
   .btn {
-    &.secondary {
-      background: $blue;
-      display: table;
-      margin: 0 auto;
-    }
+    display: table;
+    margin: 0 auto;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss]: What are these overrides used for?

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss (hunk 1 above)
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss]: Oh I was just trying to get all the elements in this div to be inline-blocks so they display next to each other. But I can update to not use a global selector. Sorry!

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss (hunk 2 above)
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss]: @weerd showed me this trick to center the button. It allows you to center it without specifying a width. The button will always be centered but allow for a variable length of text.

**Comment by DFurnes** [comment]
:+1: Looks good!

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss (hunk 2 above)
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_modal-crop.scss]: Ohhh, nice. Maybe we should make a mix-in for that at some point.

**Pull request merged**